plugins/index.md

```diff
@@ -19,7 +19,7 @@ Plugins are tools to extend and modify the functionality of Foreman. The core of
 		</a>
 	</div>
 	<div class='col-md-3 center'>
-		<a href="plugins/foreman_chef/0.1" class="btn-doc btn">
+		<a href="plugins/foreman_chef/0.2" class="btn-doc btn">
```

Comment:
Other plugins have a landing page where users can select the version, did you want one for Chef here so users can still get to 0.1?